homeassistant/components/knx/climate.py

```diff
@@ -294,9 +297,15 @@ def hvac_modes(self) -> Optional[List[str]]:
     async def async_set_hvac_mode(self, hvac_mode: str) -> None:
         """Set operation mode."""
         if self.device.supports_on_off and hvac_mode == HVAC_MODE_OFF:
-            await self.device.turn_off()
+            if not self._on_off_inverted:
+                await self.device.turn_off()
```

Comment:
I would expect this option to be handled by the library so the correct api interfaces can still be used.

Reply:
I agree. I will make a PR into the xknx library and then change it in another home-assistant PR.

Reply:
I don't think we should add this option until the library is updated.

Reply:
The change is independent of the XKNX library. When the new version of the library is available, I will make a new PR and adapt it to the library.

Reply:
Can we accept this change now, or should we wait for the library to be updated?

Reply:
The xknx library is updated by now: https://github.com/XKNX/xknx/pull/225 , but we would have to wait for the next release. 

The attribute is called `on_off_invert` (not "on_off_inverted") there.